spec/label_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../modules/label'
+
+describe Label do
+  context 'Testing Label class' do
+    label = Label.new 'Test', 'red'
+
```

Comment:
You need to add `game = Game.new 'multiplayer', '2022-03-22', '1998-12-1'`
For you tests to pass